docker/keycloak/Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM jboss/keycloak:10.0.1
 
-ARG JAR_FILE=target/*.jar
-
 USER root
 
 COPY . /project
-RUN cd /project && ./mvnw clean package && mv ${JAR_FILE} /opt/jboss/keycloak/standalone/deployments/app.jar
+RUN cd /project && ./mvnw clean package
+
+USER 1000
```

Comment:
I'm new to multi-stage builds, but is this line necessary here?

Reply:
You are correct. Setting the user there serves no purpose. PR updated.